hazelcast-sql/src/test/java/com/hazelcast/jet/sql/impl/connector/map/SqlJoinTest.java

```diff
@@ -693,6 +693,35 @@ public void test_leftJoinOnPrimitiveKey() {
         );
     }
 
+    @Test
```

Comment:
What do you think about parametrizing test by join type? We could remove the duplication this way. Also, I guess, this test would have to be split into ~ `SqlInnerJoinTest` and parametrized ~ `SqlOuterJoinTest` as parameter would not be used by the former.